addons/auth_oauth/data/auth_oauth_data.xml

```diff
@@ -16,7 +16,7 @@
             <field name="scope">public_profile,email</field>
             <field name="validation_endpoint">https://graph.facebook.com/me</field>
             <field name="data_endpoint">https://graph.facebook.com/me?fields=id,name,email</field>
-            <field name="css_class">fa fa-fw fa-facebook-square</field>
+            <field name="css_class">fa fa-fw fa-facebook</field>
```

Comment:
Since you remove the CSS, you remove also the color, same for the Odoo one. But I guess it was already the case for the google one. I feel it's better with the colors. For facebook, we have the text-facebook class. We could add one for google and odoo and use them. What do you think? 

If you say it's better without colors, we'll do as you wish :+1:

Reply:
For google we couldn't use a single plain color since it contains multiple. But while doing so I went a bit further and implemented the icons as svg to better match these company guidelines (eg. https://developers.google.com/identity/branding-guidelines)